feature-libs/my-account/organization/assets/translations/en/cost-center.i18n.ts

```diff
@@ -1,37 +1,60 @@
 export const costCenter = {
-  header: 'Cost centers',
-  disabled: '(disabled)',
+  header: 'All cost centers ({{count}})',
+
   code: 'Code',
+  active: 'Status',
   name: 'Name',
   currency: 'Currency',
   unit: 'Parent Unit',
+  actions: '',
+
+  byName: 'Sort by name',
+  byCode: 'Sort by code',
+  byUnitName: 'Sort by unit',
+
+  disable: {
+    confirm: 'Disable',
+  },
 
   messages: {
-    enabled:
-      'When you disable the cost center, the related data will be disabled as well. ',
-    disabled: 'You cannot edit a disabled Cost Center.',
-    disabledUnit: 'You cannot enable cost center if unit is disabled.',
     deactivate: 'Are you sure you want to disable this cost center?',
-    deactivateHeader: 'Disable Cost Center',
+    confirmEnabled: 'Cost Center enabled successfully',
+    confirmDisabled: 'Cost Center disabled successfully',
+  },
+
+  title: 'Details Cost Center',
+  subtitle: 'Cost Center: {{ item.name }}',
+
+  details: {
+    title: 'Details Cost Center',
+    subtitle: 'Cost Center: {{ item.name }}',
+  },
+
+  edit: {
+    title: 'Edit Cost Center',
+    subtitle: 'Cost Center: {{ item.name }}',
+  },
+
+  create: {
+    title: 'Create Cost Center',
+    subtitle: '',
   },
 
   budget: {
     link: 'Budgets',
-    header: 'Budgets in {{code}}',
-    assign: 'Assign budgets',
-    assignHeader: 'Assign budgets in {{code}}',
-    back: 'Back',
-    instructions: {
-      check: 'To assign a budget to this cost center, select its check box.',
-      uncheck: 'To unassign a budget, clear its check box.',
-      changes: 'Changes are saved automatically.',
-    },
   },
 };
 
-export const costCenterAssignBudgets = {
-  name: 'Name',
-  code: 'Code',
-  amount: 'Amount',
-  dateRange: 'Start - End',
+export const costCenterAssignedBudgets = {
+  title: 'Manage budgets',
```

Comment:
```suggestion
  title: 'Assigned budgets',
```

Reply:
UX teams wants this to become "Manage". It might affect the route as well, but i'll hold with that bigger change for now, as these are smaller text changes.